server/migrations/community/1ab5b02ce532_version_author_name_to_user_id.py

```diff
@@ -1,7 +1,7 @@
 """Migrage project version author name to user.id
 
 Revision ID: 1ab5b02ce532
-Revises: 57d0de13ce4a
+Revises: 1c23e3be03a3
```

Comment:
Did we change something? Need probably merge with current develop.

Reply:
No, it just doesn't match the line 14 in the file.